docs/wiki/research/eODS.md

```diff
@@ -0,0 +1,277 @@
+After performing the calculations, the final outcome in both cases was exactly the same, so, theoretically, the protocol would be better off cutting out the middleman, and drastically reduce staking rewards and cap total ETH staked to 6.25M.
+
+The scope of the argument above was not to advocate for reducing staking rewards, nor for capping total staked ETH, but to point out a key property that a well-functioning staking system should have: 
+ 
+>**Delegators should be doing something that actually matters** - *Vitalik Buterin*
+
+### Delegators role under eODS
+If Delegators were to have a **meaningful role**, what would that be, and how can the protocol **incentivize** that role selection?
+
+Two possible solutions[^8] arise, under Operator-Delegator separation:
+1. **The curation of operator set**: Opinionated delegators may decide to choose between different operators based on e.g., fees or reliability.
+2. **The provision of light services**: The delegators may be called upon to provide non-slashable, yet critical services, like:
+   - input their view into censorship-resistance gadgets such as inclusion lists or multiplicity gadgets.
+   - sign off on their view of the current head of the chain, as alternative signal to that of the bonded Gasper operators. 
+
+**Incentivizing the Delegator role**: 
+
+Delegators under this model do not contribute to the economic security of FFG, i.e. Delegators do not partake in Finality (non-slashable stake), but they are able to surface discrepancies in the gadget’s functioning. Their services can be compensated by re-allocated aggregated issuance.
+
+## The layers of Operator-Delegator Separation
+eODS taken in isolation, enshrines the separation, but brings no improvement to the existing market structure:
+| 1D-eODS   |           |
+| --------- | --------- |
+| OPERATOR  | DELEGATOR |
+| slashable | slashable |
+
+In order for eODS to be relevant, it must be considered holistically, together with other relevant, proposed, protocol changes:
+
+* Capping penalties:
+    | 1D-eODS + capping penalties |               |
+    | --------------------------- | ------------- |
+    | OPERATOR                    | DELEGATOR     |
+    | slashable                   | non-slashable |
+
+    In this model, by capping the slashing and leaking penalties to only the operator’s stake, assets of delegators are no longer at risk. However, Barnabe argues[^4] that the role of Delegator is not very clear under 1D-eODS, because of the following reasons:
+    * Delegators in the two-tiered staking model are unlike delegators of current LSPs, who bear the slashing risk.
+    * Some agents would wish to delegate their assets to “two-tier operators” and subject themselves to the slashing conditions, in search for yield.
+    * Some agents would wish to not operate small node    services themselves, yet participate in their provision by delegating operations instead.
```

Comment:
Small node services is discussed here for the first time, may be worth introducing it above

Reply:
added new heading above:
## The two-tier staking approach to SSF
In the [SSF](/docs/wiki/research/SSF.md) research discourse, technical limitations associated with per slot BLS signatures aggregation generates the following paradigm:
> We need to move away from the concept that every participant signs in every slot - Vitalik Buterin

In the above context, there's value in the proposal to **split staking** in two tiers of participants:[^23]
  * **A high-complexity tier** called in to act every slot but has only ~ 10,000 participants, providing **Heavy node services**, and
  * **A lower-complexity tier** only called up to participate occasionally, providing **Small(light) node services** with low computational overhead, hardware or technical know-how requirements.

The providers of heavy node services would be subject to slashing, but also highly rewarded for participating in the protocol's Finality gadget, while the providers of small node services would be entitled to lower rewards, but could be entirely exempt from slashing (do not actively participate in each slot), or could opt in to temporarily (i.e. for a few slots) become subject to slashing.

Reply:
I wouldn't emphasise complexity as the difference between heavy and light, it's really more about capital requirements. In fact I weakly think light node services may be more "complex", in that surfacing transactions for a CR gadget for instance means you want to have good networking to learn about transactions. Meanwhile participating in FFG properly "only" requires a full node basically.